ynput/OpenPype
# Use colorspace data when creating thumbnail.

## Changelog Description
Thumbnail creation was not using the colorspace data to correct convert.

## Testing notes:
1. Create a render or review.
2. Publish and verify thumbnail is correct color.

### Hosts
    - [ ] Render
    - [x] Render
    - [ ] Render
    - [ ] Render
    - [ ] Render
- [x] 3dsmax
    - [x] Render
    - [ ] Render
    - [ ] Render
    - [ ] Render
    - [ ] Render
- [ ] AE
    - [ ] Render
    - [ ] Render
    - [ ] Render
    - [ ] Render
 
This PR is now developed by @jakubjezek001
